specs/latest/1.0/index.html

```diff
@@ -395,6 +395,55 @@ <h3><a name="THE_DRAWING_BUFFER">The Drawing Buffer</a></h3>
         </p>
     </div>
 
+    <dl class="methods">
+        <dt class="idl-code">undefined drawingBufferStorage(GLenum sizedFormat, unsigned long width, unsigned long height);
+        <dd>
+            Respecify the format and size of the drawing buffer,
+            similar to <code>renderbufferStorage</code> as used on Renderbuffers.
+
+            Clearing behavior is equivalent to setting <code>HTMLCanvasElement.width</code> and <code>HTMLCanvasElement.height</code>, followed by changing the drawing buffer format.
+            This method respects <code>WebGLContextAttributes.antialias</code>.
+
+            <div class="note">
+                <code>drawingBufferStorage</code> allows for efficiently respecifying width and height together simultaneously.
+                With <code>HTMLCanvasElement.width</code> and <code>HTMLCanvasElement.height</code>,
+                setting one and then the other can incur intermediate reallocations, though User Agents do try to optimize these out.
+            </div>
+
+            If <code>drawingBufferStorage</code> is successful, the <code>drawingBufferFormat</code> becomes <code>sizedFormat</code>.
+            <code>drawingBufferWidth</code> and <code>drawingBufferHeight</code> become <code>width</code> and <code>height</code> respectively.
+            Operations on the default framebuffer therefore behave as if operating on a non-default framebuffer with an attachment of <code>sizedFormat</code>.
+            E.g. after <code>drawingBufferStorage(gl.RGBA16F, 1, 1)</code>, because the framebuffer is "floatish",
+            readPixels's main format/type pair is <code>RGBA</code>/<code>FLOAT</code> instead of <code>RGBA</code>/<code>UNSIGNED_BYTE</code>.
+
+            If <code>WebGLContextAttributes.alpha</code> is <code>false</code>, generate <code>INVALID_OPERATION</code>.
+
+            <code>RGBA8</code> is always supported for <code>sizedFormat</code>.
+            Additionally, the following enums are supported if they are currently valid for <code>renderbufferStorage</code>:
+            <ul>
+                <li><code>RGBA8</code>
+                <li><code>SRGB8_ALPHA8</code> (<code>0x8C43</code>) (E.g. from EXT_sRGB)
+                <li><code>RGBA16F</code> (<code>0x881A</code>) (E.g. from EXT_color_buffer_half_float)
+            </ul>
+            If <code>sizedFormat</code> is not currently supported, generate <code>INVALID_ENUM</code>.
+
+            <div class="note">
+                <p>
+                    In WebGL 1.0, <code>SRGB8_ALPHA8</code> requires the extension <code>EXT_sRGB<code>.
+                    In WebGL 2.0, <code>SRGB8_ALPHA8</code> does not require an extension.
+                </p>
+                <p>
+                    In WebGL 1.0, <code>RGBA16F</code> requires the extension <code>EXT_color_buffer_half_float<code>.
+                    In WebGL 2.0, <code>RGBA16F</code> requires the extension <code>EXT_color_buffer_float<code>.
```

Comment:
EXT_color_buffer_half_float's legal in WebGL 2.0 contexts now - to support iOS devices specifically. It would be better to say that WebGL 2.0 requires either EXT_color_buffer_float or EXT_color_buffer_half_float.